cupboard_app/views.py

```diff
@@ -166,6 +171,100 @@ def get(self, request: Request) -> Response:
         return Response(serializer.data)
 
 
+@extend_schema(tags=['List_Item'])
+class ListItemAPIView(APIView):
+    missingMsg = "Required value misssing from sent request,"
```

Comment:
Same as comment before about variable name formatting. Also update the "misssing" typo